go/libkb/merkle_client.go

```diff
@@ -468,13 +468,13 @@ func computeSetBitsBigEndian(x uint) []uint {
 	return ret
 }
 
-func computeLogPatternMerkleSkips(startSeqno keybase1.Seqno, endSeqno keybase1.Seqno) []uint {
-	end := uint(endSeqno)
-	var ret []uint
-	diff := end - uint(startSeqno)
-	if diff <= 0 {
+func computeLogPatternMerkleSkips(startSeqno keybase1.Seqno, endSeqno keybase1.Seqno) (ret []uint) {
```

Comment:
I would modify this method's signature so it returns an error if start > end. It seems no skips is a plausible possibility here, so just to be on the safe side. It is used a single time, should be quick. This way you can also update the test for this specific case.